app/subscriber/src/components/media-type-filters/constants/MediaFilterTypes.ts

```diff
@@ -0,0 +1,8 @@
+export enum MediaFilterTypes {
```

Comment:
Removed `HomeFilterTypes` and add `MediaFilterTypes`. Used in switch cases in `MediaFilter`

Reply:
I'm worried this is going back to hardcoded magic strings for media type filters again.  Kyle recently refactored much of this to use our dynamically managed list.

Reply:
I'll transfer that over to the newly implemented way - they differ slightly from the mockup but I don't think it should matter

Reply:
Will ping once updated - fixing a few things

Reply:
this feels like a step backwards.  Since media types are essentially a list which the user can change we should completely avoid having magic strings which match the current values in the list.